app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,3 @@
+class BookingsController < ApplicationController
+  has_many :reviews
```

Comment:
Un user peut avoir plusieurs review mais un booking ne peut avoir qu'un seul review
relation 1:1 ?

Reply:
change  to models  in last commit

Reply:
change has many to models  but . i think its . has_many : reviews through :user ?